manifests/config.pp

```diff
@@ -179,20 +179,27 @@
       }
     }
     'systemd': {
-      systemd::unit_file {'prometheus.service':
+      file { '/etc/systemd/system/prometheus.service':
+        mode    => '0644',
+        owner   => 'root',
+        group   => 'root',
```

Comment:
same with mode/group. `group = root` will particularly make this fail on platforms where gid 0 is not `root` (but e.g. `wheel`, although arguably those platforms are unlikely to run systemd).